packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.tsx

```diff
@@ -59,20 +52,19 @@ function toPercent(value: number) {
   return `${(value * 100).toFixed(1)}%`;
 }
 
-function SpanBar(props: TInnerProps) {
+function SpanBar(props: TOuterProps) {
   const {
     viewEnd,
     viewStart,
     getViewedBounds,
     color,
-    label,
     hintSide,
     onClick,
-    setLongLabel,
-    setShortLabel,
```

Comment:
So previously these functions could've been supplied externally?  Is it safe to remove them?

Reply:
were they defined to be used from tests?

Reply:
The `SpanBar` component is referenced in `SpanBarRow.tsx` and `SpanBar.test.js` but it doesn't look like the props `setLongLabel` or `setShortLabel` were used. 


<img width="1339" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/59589720/438c6aa6-2b04-46dd-83d2-d77a1b6a73c8">